packages/help-center/src/styles.scss

```diff
@@ -62,6 +63,17 @@ $head-foot-height: 50px;
 		}
 	}
 
+	.help-center-header__tab-panel {
```

Comment:
**Note:**

The designs from the mocks diverge from the tab styling of the shared component. I override tab styling here.